demo-notebooks/guided-demos/2_basic_jobs.ipynb

```diff
@@ -203,7 +203,9 @@
    "id": "31096641",
    "metadata": {},
    "source": [
-    "Now, an alternative option for job submission is to submit directly to MCAD, which will schedule pods to run the job with requested resources:"
+    "Now, an alternative option for job submission is to submit directly to MCAD, which will schedule pods to run the job with requested resources:\n",
+    "\n",
+    "NOTE: To test this demo in an air-gapped/ disconnected environment alter the training script to use a local dataset."
```

Comment:
Thinking whether to adjust the instruction also for Ray part above, not just MCAD. WDYT?

Reply:
💯 , I missed this. Great suggestion thank you